src/ui/public/timepicker/kbn_global_timepicker.html

```diff
@@ -28,5 +31,6 @@
       <i aria-hidden="true" class="fa fa-clock-o"></i>
       <pretty-duration from="timefilter.time.from" to="timefilter.time.to"></pretty-duration>
     </button>
+    <i ng-click="foward()" class="time-nav-btn fa fa-arrow-right ng-scope"></i>
   </li>
 </ul>
```

Comment:
sorry to hijack this PR for something a little broader ;)

![image](https://cloud.githubusercontent.com/assets/1833023/19169089/dc6bc0fc-8be0-11e6-80a7-413ab4363b2b.png)

With the 4 new additions, the top toolbar now looks like the above. From a UI perspective, would it make sense to better isolate the timepicker tools from the "new/save/open/share" buttons. Both are unrelated thematically. Putting a divisor in between/shading background differently/ .... might make the grouping a little clearer.  @alt74, @cjcenizal, thoughts (?)